### PR TITLE
swaks: add page

### PR DESCRIPTION
**Pull request by peterbabic**
- [x] The page is in the correct platform directory (`common/`, `linux/`,
      etc.)
- [x] The page (if new), does not already exist in the repository.
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended
      [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the
      [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage
      (if applicable).

**Version of the command being documented (if known):** v20201014.0

### CODE REVIEW

File: pages/linux/swaks.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# swaks
+
+> Swiss Army Knife SMTP, the all-purpose SMTP transaction tester.
+> More information: <https://github.com/jetmore/swaks/blob/0/doc/base.pod>.
```
Comment (navarroaxel) [inline pages/linux/swaks.md:4]: The link is a 404
```suggestion
> More information: <https://github.com/jetmore/swaks/blob/develop/doc/base.pod>.
```

File: pages/linux/swaks.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+`swaks --to {{user@example.com}} --server {{test-server.example.net}}`
+
+- Deliver a standard test email, requiring CRAM-MD5 authentication as user me@example.com. An "X-Test" header will be added to the email body. The authentication password will be prompted for if it cannot be obtained from your .netrc file:
+
+`swaks --to {{user@example.com}} --from {{me@example.com}} --auth {{CRAM-MD5}} --auth-user {{me@example.com}} --header-X-Test {{"test email"}}`
+
+- Test a virus scanner using EICAR in an attachment. Don't show the message DATA part:
+
+`swaks -t {{user@example.com}} --attach - --server {{test-server.example.com}} --suppress-data {{/path/to/eicar.txt}}`
```
Comment (navarroaxel) [inline pages/linux/swaks.md:16]: Should be absolute path?
```suggestion
`swaks -t {{user@example.com}} --attach - --server {{test-server.example.com}} --suppress-data {{path/to/eicar.txt}}`
```

File: pages/linux/swaks.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+- Test a spam scanner using GTUBE in the body of an email, routed via the MX records for example.com:
+
+`swaks --to {{user@example.com}} --body {{/path/to/gtube/file}}`
```
Comment (CleanMachine1) [inline pages/linux/swaks.md:20]:
```suggestion
`swaks --to {{user@example.com}} --body {{path/to/gtube/file}}`
```

As Axel mentioned, does it need to be absolute?

File: pages/linux/swaks.md (hunk 2 above)
Comment (CleanMachine1) [reply pages/linux/swaks.md:16]: Should there be an extra `-` between --attach and --server?

File: pages/linux/swaks.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+> Swiss Army Knife SMTP, the all-purpose SMTP transaction tester.
+> More information: <https://github.com/jetmore/swaks/blob/0/doc/base.pod>.
+
+- Deliver a standard test email to user@example.com on port 25 of test-server.example.net:
+
+`swaks --to {{user@example.com}} --server {{test-server.example.net}}`
+
+- Deliver a standard test email, requiring CRAM-MD5 authentication as user me@example.com. An "X-Test" header will be added to the email body. The authentication password will be prompted for if it cannot be obtained from your .netrc file:
+
+`swaks --to {{user@example.com}} --from {{me@example.com}} --auth {{CRAM-MD5}} --auth-user {{me@example.com}} --header-X-Test {{"test email"}}`
```
Comment (EmilySeville7cfg) [inline pages/linux/swaks.md:12]:
```suggestion
`swaks --to {{user@example.com}} --from {{me@example.com}} --auth {{CRAM-MD5}} --auth-user {{me@example.com}} --header-X-Test "{{test_email}}"`
```

We use quotes outside placeholders.

File: pages/linux/swaks.md (hunk 3 above)
Comment (EmilySeville7cfg) [inline pages/linux/swaks.md:20]:
```suggestion
`swaks --to {{user@example.com}} --body {{/path/to/gtube_file}}`
```

**Review by EmilySeville7cfg — CHANGES_REQUESTED** [review]

File: pages/linux/swaks.md (hunk 3 above)
Comment (marchersimon) [reply pages/linux/swaks.md:20]:
```suggestion
`swaks --to {{user@example.com}} --body {{path/to/gtube_file}}`
```

File: pages/linux/swaks.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
+# swaks
+
+> Swiss Army Knife SMTP, the all-purpose SMTP transaction tester.
+> More information: <https://github.com/jetmore/swaks/blob/develop/doc/base.pod>.
+
+- Deliver a standard test email to user@example.com on port 25 of test-server.example.net:
```
Comment (marchersimon) [inline pages/linux/swaks.md:6]:
```suggestion
- Deliver a standard test email to `user@example.com` on port 25 of `test-server.example.net`:
```

File: pages/linux/swaks.md [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+
+`swaks --to {{user@example.com}} --server {{test-server.example.net}}`
+
+- Deliver a standard test email, requiring CRAM-MD5 authentication as user me@example.com. An "X-Test" header will be added to the email body. The authentication password will be prompted for if it cannot be obtained from your .netrc file:
```
Comment (marchersimon) [inline pages/linux/swaks.md:10]: I'd say this sentence is not that essential.
```suggestion
- Deliver a standard test email, requiring CRAM-MD5 authentication as user `me@example.com`. An "X-Test" header will be added to the email body:
```

File: pages/linux/swaks.md [hunk 7]
```diff
@@ -0,0 +1,24 @@
...
+
+- Test a virus scanner using EICAR in an attachment. Don't show the message DATA part:
+
+`swaks -t {{user@example.com}} --attach - --server {{test-server.example.com}} --suppress-data {{path/to/eicar.txt}}`
+
+- Test a spam scanner using GTUBE in the body of an email, routed via the MX records for example.com:
+
+`swaks --to {{user@example.com}} --body {{path/to/gtube_file}}`
+
+- Deliver a standard test email to user@example.com using the LMTP protocol via a UNIX domain socket file:
```
Comment (marchersimon) [inline pages/linux/swaks.md:22]:
```suggestion
- Deliver a standard test email to `user@example.com` using the LMTP protocol via a UNIX domain socket file:
```

**Review by marchersimon — APPROVED** [review]

**Review by EmilySeville7cfg — APPROVED** [review]

**Comment by EmilySeville7cfg** [comment]
Merged, thanks! 👍🏻

**Pull request merged**